docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: GX Overview
+---
+
+import PythonVersion from '/docs/components/_python_version.mdx';
+
+Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, GX applies Expectations to data instead of code. For example, you could define an Expectation that a column contains no null values, and GX would run that Expectation against your data, and report if a null value was found.
+
+## Before you start
+
+Before you've even installed GX, working with GX starts with data.  Specifically, data you care about enough for quality to be important.  We call your data, in its original format, your **Source Data**.  The combination of your data in its original format and the location at which it is stored is your **Source Data System**.
+
+Your Source Data System could be CSV files in a folder, or a PostgreSQL database hosted on AWS, or any combination of numerous formats and environments.  Regardless of what your Source Data System is or where it resides, your next step in assuring its quality is implementing GX.
+
+## Getting started with Great Expectations
```

Comment:
```suggestion
## Get started with Great Expectations
```
I recommend that we avoid using gerunds in headings.

![image](https://github.com/great-expectations/great_expectations/assets/104205257/03ae93ae-c790-42e9-894e-73f76d04dfc0)